ui_framework/components/button/link_button.test.js

```diff
@@ -17,6 +17,74 @@ describe('KuiLinkButton', () => {
     });
   });
 
+  describe('HTML attributes', () => {
```

Comment:
I would fold all of these into one single test (without a `describe`), e.g.

```js
test('renders HTML attributes as-is', () => {
})
```

or something like that. I've gradually made my snapshots tests a little bit "broader" instead of being super-focused (it's kinda "unlearning TDD"). The point is that the snapshot together with the test is communicating what's allowed. It also creates "less noise" imo.